sdk/FEATURES.md

```diff
@@ -197,6 +195,5 @@ An exit handler is a component that always executes, irrespective of success or
 
 [ParallelFor]: https://github.com/tektoncd/pipeline/issues/2050
 [VarSub]: https://github.com/tektoncd/pipeline/issues/2322
```

Comment:
VarSub was also addressed in KFP-tekton, is it?

Reply:
VarSub is not resolved, the issue is being redirected to a different one with the same feature requirement. I updated the issue link to the new issue.